Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -24,11 +26,24 @@ public class NoOpRunner : ISubject
 
         public event EventHandler<MessageDto> OnMessageReceived;
 
-        public PlatformsContainer PlatformsContainer { get; private set; }
+        public PlatformsContainer PlatformsContainer 
+        {
+            get => GameState.Platforms; 
+            set => GameState.Platforms = value; 
+        }
 
-        public Player Player { get; private set; }
+        public Player Player
+        {
+            get => GameState.Player;
+            set => GameState.Player = value;
+        }
```

Comment:
These changes broke host-client sync, fix `ClientHandleMessage()` to set `GameState` properly, it now triest to set `GameState.Player` etc with `GameState == null`